charts/ingress-nginx/templates/tarantool-daemonset.yaml

```diff
@@ -38,8 +38,8 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
     spec:
-    {{- if .Values.controller.wallarm.imagePullSecrets }}
-      imagePullSecrets: {{ toYaml .Values.controller.wallarm.imagePullSecrets | nindent 8 }}
+    {{- if .Values.imagePullSecrets }}
+      imagePullSecrets: {{ toYaml .Values.imagePullSecrets | nindent 8 }}
```

Comment:
we need to announce this change in the change log at release, because some customers can already have `.Values.controller.wallarm.imagePullSecrets` in use